foundry.toml

```diff
@@ -20,6 +20,7 @@
 
 [rpc_endpoints]
   base = "${BASE_RPC_URL}"
+  mainnet = "${MAINNET_RPC_URL}"
```

Comment:
What is this used for? I think we should need RPC urls for two networks: `BASE` and `BASE_GOERLI`

Reply:
nevermind, thought we were deploying to mainnet as well. Removed it in [5b64a2d](https://github.com/showtime-xyz/creator-tokens/pull/40/commits/5b64a2d85b26c831690a7e67d65a0adcc2610c29)